gwvolman/lib/dataone/metadata.py

```diff
@@ -106,8 +106,9 @@ def create_resource_map(self, pid, scimeta_pid, sciobj_pid_list):
         :return: The ORE object
         :rtype: d1_common.resource_map.ResourceMap
         """
-
-        ore = ResourceMap(base_url=self.coordinating_node)
+        if self.coordinating_node.endswith("v2"):
```

Comment:
As far as I can tell, `coordinating_node` always ends with `v2`. If it doesn't, you'd need to set `ore_endpoint` to `coordinating_node` in an else. Also, it's not obvious to me why we strip instead of just removing the v2 from the initial format: 

https://github.com/whole-tale/gwvolman/blob/1e0f1d3cf2e37542e6a11abad56a7d04a11b3b59/gwvolman/lib/dataone/publish.py#L44



I gather that the D1 `ResourceMap` adds the v2 regardless, but that other calls require it?


https://github.com/DataONEorg/d1_python/blob/a49a5430edbfdc9415c9c1fa6026532ae91966ee/lib_common/src/d1_common/resource_map.py#L619

Reply:
Ooops that was dumb to leave out. Fixed. We use it when querying for supported mimetypes...

So we can either remove `v2` completely and re-add it when making that call, or leave it as is and remove the v2 when creating the resource map; it doesn't make much of a difference to me.